app/code/Magento/Inventory/Model/StockIndexTableNameResolver.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Inventory\Model;
```

Comment:
Now we have separate module related indexation functionality
Need to move Interface and implementation to `InventoryIndexer`

Reply:
Done.